R/plots.R

```diff
@@ -27,90 +27,91 @@
              ggplot2::geom_bar(stat = "identity", position="dodge") +
              ggplot2::facet_wrap(~group)
-     },
-     ordinal = {
-      switch(x_axis,
-        group = {
-          ggplot2::ggplot(df, ggplot2::aes_string(x='stratum', y='weight', col='fold'))+
-               ggplot2::geom_point() +
-               ggplot2::geom_line() +
-               ggplot2::scale_x_continuous(breaks= scales::pretty_breaks())
-        }, 
-        stop(sprintf("x_axis = '%s' not implemented.", x_axis))
-      )},
-      stop(sprintf("Invalid scale='%s'.", scale))
+         },
+         ordinal = {
+           switch(x_axis,
+                  group = {
+                    ggplot2::ggplot(df, ggplot2::aes_string(x='stratum', y='weight', col='fold'))+
+                      ggplot2::geom_point() +
+                      ggplot2::geom_line() +
+                      ggplot2::scale_x_continuous(breaks= scales::pretty_breaks())
+                  }, 
+                  stop(sprintf("x_axis = '%s' not implemented.", x_axis))
+           )},
+         stop(sprintf("Invalid scale='%s'.", scale))
   ) 
 }  
 
 
 plot_decisionboundary <- function(x, df, x_axis, scale) {
-
+  
   if (x_axis != "covariate")
     warning(sprintf("x_axis='%s' not implemented.", x_axis))
- 
+  
   xdf <- x@df[ !is.na(x@df$pvalue), ]
-
+  
   minp <- 10 ^ (-ceiling(log10(nrow(xdf))) - 6)
-                             
+  
   xdf$pvalue[ xdf$pvalue < minp] <- minp
   xdf$rkcv <- rank(xdf$covariate, ties.method = "min")
-    
+  
   minrk      <- by(xdf, xdf$group, function(v) min(v$rkcv))
   pthresh    <- by(xdf, list(xdf$group, xdf$fold), function(v) {
-                        wh <- (v$adj_pvalue <= x@alpha)
-                        if (!any(wh)) minp else max(v$pvalue[wh], na.rm = TRUE) 
-                        })
+    wh <- (v$adj_pvalue <= x@alpha)
+    if (!any(wh)) minp else max(v$pvalue[wh], na.rm = TRUE) 
+  })
   
   df$minrk   <- minrk[as.character(df$group)]
   df$pthresh <- pthresh[cbind(as.character(df$group), as.character(df$fold))]
-
+  
   # add another bin for plotting
   toadd <- subset(df, stratum == nlevels(groups_factor(x)))
   toadd$minrk   <- rep(nrow(xdf), nrow(toadd))  
   df <- rbind(df, toadd)
-    
+  
   switch(scale,
          nominal = {
            stop("Not yet implemented.")
          },
          ordinal = {
-             ggplot2::ggplot(xdf, ggplot2::aes(x = rank(covariate), y = -log10(pvalue))) +
-               ggplot2::geom_hex( bins = 100) +
-               ggplot2::geom_step( ggplot2::aes(x = minrk, y = -log10(pthresh), col = fold), data = df, direction = "hv")
+           ggplot2::ggplot(xdf, ggplot2::aes(x = rank(covariate), y = -log10(pvalue))) +
+             ggplot2::geom_hex( bins = 100) +
+             ggplot2::geom_step( ggplot2::aes(x = minrk, y = -log10(pthresh), col = fold), data = df, direction = "hv")
          },
          stop(sprintf("Invalid scale='%s'.", scale))
   ) 
-}  
+}  
```

Comment:
Please try to fix the white space in this file so that it gets easier to review

Reply:
In the new commit, white space changes are finally not being displayed anymore (I was aware of this and had a few tries before)